kivy/tests/async_common.py

```diff
@@ -1,23 +1,26 @@
-import asyncio
 import random
 import time
+import os
 from collections import deque
 
 from kivy.tests import UnitTestTouch
 
 __all__ = ('UnitKivyApp', 'async_sleep')
 
 
+KIVY_EVENTLOOP = os.environ.get('KIVY_EVENTLOOP')
```

Comment:
This and everywhere else it should be `kivy_eventloop= os.environ.get('KIVY_EVENTLOOP', 'asyncio')`. This is so that the the asyncio tests will be run if the proper `pytest-asyncio` library is installed. 

Otherwise, there's no way to run these asyncio tests because if we set `KIVY_EVENTLOOP=sync`, then these tests won't run, but if we set it to `asyncio`, then all the other tests won't work because they don't expect this. This way, only the async tests will run as needed.

We could make the default `trio`, which I honestly would prefer because it's less buggy, but I don't want to add `trio` as a dependency in travis, since `asyncio` is preinstalled already.

Reply:
> but if we set it to asyncio, then all the other tests won't work because they don't expect this.

I agree with you. I kinda forgot about it.

> We could make the default `trio` ... since asyncio is preinstalled already.

I feel like the default should be `asyncio` since it's preinstalled as you stated.